.github/workflows/trivy-scan.yml

```diff
@@ -0,0 +1,45 @@
+name: Trivy Scan
+on:
+  pull_request:
```

Comment:
Would this flag vulnerabilities that are not relevant to PR but just got reported in trivy DB by the time of the scan?

Reply:
Yes - there's no ability to scan only changed files. But this isn't (yet) a required workflow, so new vulnerabilities wouldn't prevent a PR from being merged.